app/version-check/server/index.js

```diff
@@ -24,18 +25,24 @@ const addVersionCheckJob = () => {
 
 
 Meteor.startup(() => {
-	Meteor.defer(() => checkVersionUpdate());
+	Meteor.defer(() => {
+		if (settings.get('Register_Server') && settings.get('Update_EnableChecker')) {
+			checkVersionUpdate();
+		}
+	});
 });
 
-settings.get('Register_Server', (key, value) => {
-	if (value && SyncedCron.nextScheduledAtDate(jobName)) {
+settings.get(/(Register_Server)|(Update_EnableChecker)/, _.debounce(() => {
```

Comment:
you don't need a capturing group =)
```suggestion
settings.get(/Register_Server|Update_EnableChecker/, _.debounce(() => {
```